tools/bench_fp8_gemm.py

```
@@ -0,0 +1,611 @@
+    fn.argtypes = [
+        ctypes.c_void_p, ctypes.c_void_p, ctypes.c_void_p, ctypes.c_void_p,
+        ctypes.c_int, ctypes.c_int, ctypes.c_int, ctypes.c_int, ctypes.c_int,
+        ctypes.c_int, ctypes.c_int, ctypes.c_int,
+        ctypes.POINTER(ctypes.c_float), ctypes.c_void_p,
+    ]
+    return fn
+
+
+def _cutlass_config_id(M):
+    """M-bucket dispatch matching gemm_fp8_blockwise_tuned.h defaults."""
+    if M <= 16:   return 0  # 64×128 Pingpong
```

Ruff (0.15.1)

[error] Multiple statements on one line (colon)

(E701)